constraints.txt

```diff
@@ -0,0 +1,92 @@
+aniso8601==3.0.2
```

Comment:
How about naming it something like `requirements-lock.txt`, to comply with other similar systems?

Reply:
wait, this file was not suppose to be in this PR. Pleas move the discussion to : https://github.com/raiden-network/raiden/pull/2469